src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/XamlSchemaContext.cs

```diff
@@ -542,6 +542,9 @@ private string GetCompatibleNamespace(string oldNs)
 
         public virtual XamlType GetXamlType(Type type)
         {
+            if (type == null)
```

Comment:
This one would succeed in XamlLanguage.TypeAlias(type) but throw in GetXamlType, By adding an explicit throw here, we make sure to throw early, with the right stacktrace and the right parameter name.